evm_arithmetization/src/generation/prover_input.rs

```diff
@@ -307,6 +308,15 @@ impl<F: Field> GenerationState<F> {
             ))
         }
     }
+    /// Returns a non-jumpdest proof for the address on the top of the stack. A
+    /// non-jumpdest proof is the clossest address to address on the top of
+    /// the stack.
+    fn run_next_non_jumpdest_proof(&mut self) -> Result<U256, ProgramError> {
+        let code = self.get_current_code()?;
```

Comment:
Do you think getting the code could benefit from memoization, i.e. do the conversion from U256s to u8s once and store it in a `HashMap` for quick lookup oin subsequent calls? To avoid some memory churn?

Reply:
No. Because it's only called once, when the transaction must abort because of an invalid jump.